### ISSUE TITLE
Poetry

### ISSUE THREAD

**Issue by Trutifruti** [dataforgoodfr/energetic-stress-production#31]
Would it be an option to use poetry for version control of all the used packages ? 

The requierents.txt isn't exhaustive as of now. We could make a script to have it update automatically, but poetry makes the package management and addition of new packages very easy. 
With 3 terminal commands we can 
1. add new packages.
2. update all packages
3. synchronize your local venv with the list of requirements

**Comment by antoinetavant** [comment]
Yes, the requierment.txt is not used currently, it should be removed from the project.
The project is supposed to use Hatch, with is somewhat similar but different from Poetry.

But recently, I thought of using only UV instead of hatch. 
https://docs.astral.sh/uv/
The last release of UV is beast. It does all we need.

For more discussion on this topic, I recommend this article : https://alpopkes.com/posts/python/packaging_tools/
